web/profiles/custom/yalesites_profile/config/sync/views.view.block_content.yml

```diff
@@ -21,7 +21,7 @@ display:
     display_plugin: default
     position: 0
     display_options:
-      title: 'Custom block library'
+      title: 'Global Blocks'
```

Comment:
I think we are changing all of the 'globals' to 'reusable'. I'm guessing you are already working on this.